common/tile.h

```diff
@@ -170,3 +170,16 @@ bool tile_virtual_check(const tile *vtile);
 
 bool tile_set_label(struct tile *ptile, const char *label);
 bool tile_is_placing(const struct tile *ptile);
+
+/**
+  Contains some information about a tile as human readable translated
+  string.
+ */
+struct tile_info {
```

Comment:
I'm wondering if we shouldn't use a constructor instead of `tile_get_info`. That is,
```suggestion
struct tile_info {
    explicit tile_info(const tile *ptile);
```
This would be used as `tile_get_info_text(tile_info(ptile), ...)`.

Reply:
I wasn't aware of struct constructors. But I like it, done.